### PR TITLE
Slave page

### PR DESCRIPTION
**Pull request by Hornwitser**
Add per slave page to the web UI displaying the console for the slave and the instances assigned to it. Also adds console view to the master page.

### CODE REVIEW

**Review by psihius — APPROVED** [review]

File: packages/web_ui/src/components/SlaveViewPage.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	}
+
+	return <PageLayout nav={nav}>
+		<PageHeader
+			className="site-page-header"
+			title={slave.name || slaveId}
+		/>
+
+		<Descriptions bordered size="small" column={{ xs: 1, sm: 2, xl: 4 }}>
+			<Descriptions.Item label="Name">{slave["name"]}</Descriptions.Item>
```
Comment (Danielv123) [inline packages/web_ui/src/components/SlaveViewPage.jsx:46]: Is there a reason why you use bracket syntax here instead of . syntax?

**Review by Danielv123 — APPROVED** [review]
Pretty much the exact same functionality as I have in my local branch. Can't believe I have forgotten to make a PR for it for like 3 months now... This was cleaner anyways though.

**Comment by psihius** [comment]
> Pretty much the exact same functionality as I have in my local branch. Can't believe I have forgotten to make a PR for it for like 3 months now... This was cleaner anyways though.

This is why you make "in progress" pull requests :D

File: packages/web_ui/src/components/SlaveViewPage.jsx (hunk 1 above)
Comment (Hornwitser) [reply packages/web_ui/src/components/SlaveViewPage.jsx:46]: It's a hint that it's a plain object, like deserialized JSON.

Comment (Danielv123) [reply packages/web_ui/src/components/SlaveViewPage.jsx:46]: Ah, I see. I will try to keep to the same pattern then.

**Pull request merged**